### PR TITLE
Mention CF_ZONE_ID in environment variables

### PR DESCRIPTION
**Pull request by nchlswhttkr**
I've added a mention of `CF_ZONE_ID` to the section on [using environment variables](https://developers.cloudflare.com/workers/tooling/wrangler/configuration#using-environment-variables) per the feedback on #704.

The other environment variables (for account ID/email and the associated credentials) are already mentioned/covered.

### CODE REVIEW

File: src/content/tooling/wrangler/configuration/_index.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ There are two types of configuration that `wrangler` uses: global user and per p
   ```
   Note that providing authentication credentials through environment variables will override whatever credentials you configured
```
Comment (victoriabernard92) [inline src/content/tooling/wrangler/configuration/_index.md]:
```suggestion
  Note environment variables will override whatever credentials you configured in `wrangler config` or in your `wrangler.toml`. 
```

**Comment by victoriabernard92** [comment]
Since configuring zone ID and account ID are on a per project basis this is odd being in the Global User section. I think we should actually move environment variables into its own section and list out all the settings that can be set in environment variables there. Then in the Global User section just have the sentence that mentions env. variables are how to set up CI.

Side note, I am trying to brainstorm  another term would could use since environments are being overloaded term for Workers 🤔 any ideas?

**Comment by nchlswhttkr** [comment]
I think environment variables is still the term to use here, and changing it drastically might make it harder for people searching the docs with keywords.

Maybe stressing that they can be used for "overriding/injecting" configuration would help? For example, "Overriding project configuration with environment variables in the wrangler CLI".

As for the environments that a worker is previewed/published to, maybe they could be referred to as "deployment environments" or the like? Trying to differentiate it from what's happening in your terminal, and instead focusing on how it ties in with the publishing process.

**Comment by victoriabernard92** [comment]
Once we resolve merge conflicts, this lgtm

**Comment by nchlswhttkr** [comment]
Apologies for the delay, merge conflicts should be resolved now.

**Pull request merged**